src/1.18/chunk.js

```diff
@@ -112,14 +115,15 @@ module.exports = (ChunkColumn, registry) => {
   function fromNBT (tag) {
     const data = nbt.simplify(tag)
     const column = new ChunkColumn({ minY: -64, worldHeight: 384 })
-    assert(data.Status === 'full', 'Chunk is not full')
+    assert(data.Status === 'minecraft:full' || data.Status === 'full', 'Invalid chunk status, must be full: ' + data.Status)
 
     column.x = data.xPos
     column.z = data.zPos
     column.lastUpdate = data.LastUpdate.valueOf()
     column.inhabitedTime = data.InhabitedTime.valueOf()
 
     for (const section of data.sections) {
+      if (!section.block_states) continue
```

Comment:
Is this valid? What's the reasoning behind this change?

Reply:
This is also to fix saving test for the latest version. If the section doesn't have information what's the point to proceed it?

Reply:
In what case can a section not have block states ? Sounds like an invalid state that should be addressed rather than ignored